modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -19,18 +23,12 @@ GA:: General Availability
 
 | Operator | Pipelines | Triggers | CLI | Chains | Hub | {pac} | Results | Manual Approval Gate | |
```

Comment:
Outside of the scope of this PR, but noticed during review so I'll mention for any potential follow-ups: Headers in table should use sentence case like other headings, so e.g., this line and the headers a couple lines before it could be:

```
| {pipelines-title} version 8+| Component version | OpenShift version | Support status
| Operator | Pipelines | Triggers | CLI | Chains | Hub | {pac} | Results | Manual approval gate | |
```